scripts/pi-hole/js/messages.js

```diff
@@ -178,19 +166,21 @@ function delMsg(ids) {
         utils.showAlert(
           "success",
           "far fa-trash-alt",
-          "Successfully deleted " + ids.length + " message" + (ids.length > 1 ? "s" : ""),
-          ""
+          "Successfully deleted message",
+          "",
```

Comment:
To stay similar to other areas on the web interface, I'd prefer a third option, like
```suggestion
          "ID: " + id,
```